selfdrive/car/toyota/values.py

```diff
@@ -939,18 +941,21 @@ class CAR:
       b'F152642541\x00\x00\x00\x00\x00\x00',
     ],
     (Ecu.eps, 0x7a1, None): [
+      b'8965B42171\x00\x00\x00\x00\x00\x00',
       b'8965B42170\x00\x00\x00\x00\x00\x00',
       b'8965B42171\x00\x00\x00\x00\x00\x00',
       b'8965B42181\x00\x00\x00\x00\x00\x00',
       b'\x028965B0R01200\x00\x00\x00\x008965B0R02200\x00\x00\x00\x00',
       b'\x028965B0R01300\x00\x00\x00\x008965B0R02300\x00\x00\x00\x00',
     ],
     (Ecu.fwdRadar, 0x750, 0xf): [
+      b'\x018821F3301400\x00\x00\x00\x00',
```

Comment:
duplicate

Reply:
Just double checked and this looks like it's the only unique value.